src/client/pages/AssessmentSection/Title/Title.tsx

```diff
@@ -62,7 +62,7 @@ const Title: React.FC<Props> = (props) => {
             lang={i18n.language}
           />
           <DefinitionLink
-            assessmentName={assessmentName}
+            assessmentName={assessmentName.toLowerCase()}
```

Comment:
The link should use lowercase, not here

Reply:
Do you mean the `DefinitionLink` component??

Reply:
Or in the `ApiEndPoint` where the link is build?